config/src/aws-net-std__ipv6.adb

```diff
@@ -190,8 +199,9 @@ package body AWS.Net.Std is
    is
       use type C.int;
 
-      Info  : constant OS_Lib.Addr_Info_Access :=
+      Raw   : constant OS_Lib.Addr_Info_Access :=
```

Comment:
So, both constants below can be "not null OS_Lib.Addr_Info_Access :="

Reply:
I think it is superfluous because "not null" already checked as function result and could not be changed because of "constant". But .. if you prefer, Done.

Reply:
I told not about that compiler could do not do the superfluous check. I told that we do not need to write "not null" for the constants taking the data from such functions.